src/Symfony/Component/DependencyInjection/Attribute/TaggedIterator.php

```diff
@@ -11,15 +11,44 @@
 
 namespace Symfony\Component\DependencyInjection\Attribute;
 
+/**
+ * Autowires an iterator of services based on a tag name.
+ */
 #[\Attribute(\Attribute::TARGET_PARAMETER)]
 class TaggedIterator extends AutowireIterator
 {
     public function __construct(
+        /**
+         * The tag to look for to populate the iterator.
+         */
         public string $tag,
+
+        /*
```

Comment:
There is CPP in this constructor.
```suggestion
        /**
```